demisto_sdk/commands/validate/validators/ST_validators/ST111_no_exclusions_schema.py

```diff
@@ -0,0 +1,51 @@
+from typing import Iterable
+
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+ALLOWED_SECTIONS = [
+    "Connect",
+    "Collect",
+    "Optimize",
+]
+
+
+class StrictSchemaValidator(BaseValidator[ContentTypes]):
+    error_code = "ST111"
+    description = "Validate that the scheme's structure is valid, no fields excluded."
+    rationale = "Maintain valid structure for content items."
+
+    def obtain_invalid_content_items(
+        self,
+        content_items: Iterable[ContentTypes],
+    ) -> list[ValidationResult]:
+        invalid_content_items = []
+        for content_item in content_items:
+            error_message = self.get_error_message(content_item)
+            if error_message:
+                invalid_content_items.append(
+                    ValidationResult(
+                        validator=self,
+                        message=error_message,
+                        content_object=content_item,
+                    )
+                )
+        return invalid_content_items
+
+    def get_error_message(self, content_item: ContentTypes) -> str:
```

Comment:
Change this name for something more informative like `is_missing_section_fields`.